src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx

```diff
@@ -126,15 +130,11 @@ const PublicationReleasePage: NextPage<Props> = ({ release }) => {
                     <Link
                       className="govuk-!-display-none-print govuk-!-display-block govuk-!-margin-bottom-3"
                       unvisited
-                      to={`/find-statistics/${release.publication.slug}/${release.slug}`}
+                      to={`/find-statistics/${release.publication.slug}/${releaseSeriesNonLegacy?.releaseSlug}`}
```

Comment:
I think this works but naming it something like `latestReleaseSeriesItem` rather than `releaseSeriesNonLegacy` would help explain why it's being used here.